spec/Section 7 -- Response.md

```diff
@@ -244,6 +238,19 @@ discouraged.
 }
 ```
 
+### Path
+
+:: A _path field_ allows for the association to a particular field in a GraphQL
+result. This field must be a list of path segments starting at the root of the
+response and ending with the field to be associated with. Path segments that
+represent fields must be strings, and path segments that represent list indices
+must be 0-indexed integers. If the path is associated to an aliased field, the
+path must use the aliased name, since it represents a path in the response, not
+in the request.
```

Comment:
"a particular field in a GraphQL result" isn't strictly true for errors, because errors may cause the selection sets to explode, and thus the path may point to a field that _does not exist_ in the GraphQL result.

```suggestion
:: A _path entry_ allows for the association with a particular field reached
during GraphQL execution. The value for this entry must be a list of path
segments starting at the root of the response and ending with the field to be
associated with. Path segments that represent fields must be strings, and path
segments that represent list indices must be 0-indexed integers. If a path
segment is associated with an aliased field it must use the aliased name, since
it represents a path in the response, not in the request.
```